.gitignore

```diff
@@ -4,4 +4,3 @@ projects/**/Release
 projects/**/x64
 projects/**/*.user
 
-hebi/
```

Comment:
I don't think this should be removed... Should it be removed? @iamtesch

Reply:
Interesting, this wasn't intentional from my side

Reply:
I think this is an artifact of when we explicitly added the C++ repo.

However....the C++ repo downloads into `hebi-cpp`, right?  I'm not sure this should be in here?